.flake8

```diff
@@ -8,9 +8,10 @@
 # W504: line break after binary operator
 #       (Raised by flake8 even when it is followed)
 ignore = E126, E402, E129, W504
-max-line-length = 152
+max-line-length = 131
 exclude = test_import_fail.py,
-  parsl/executors/workqueue/parsl_coprocess.py
+  parsl/
```

Comment:
I think this is a mistake here: it looks like you've split the line containing `parsl/executors/workqueue/parsl_coprocess.py` into two separate lines, and now the new version says to ignore all errors in the whole of parsl!